crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -264,6 +270,9 @@ impl AnalyzeEcmascriptModuleResultBuilder {
         mut self,
         track_reexport_references: bool,
     ) -> Result<Vc<AnalyzeEcmascriptModuleResult>> {
+        let bindings = EsmBindings::new(take(&mut self.bindings));
+        self.add_code_gen(bindings);
```

Comment:
Check for self.bindings.is_empty(), e. g. commonjs modules won't have bindings at all